projects/observability/src/shared/components/topology/d3/d3-topology.ts

```diff
@@ -95,7 +98,8 @@ export class D3Topology implements Topology {
     this.hover = new TopologyHover(this.d3Util, this.domRenderer);
     this.click = new TopologyClick(this.d3Util, this.domRenderer);
     this.zoom = new TopologyZoom();
-    this.layout = this.initializeLayout(config.layoutType);
+    this.layout = this.config.customLayout ?? this.initializeLayout(this.config.layoutType);
+    this.supportGroupNode = this.config.supportGroupNode ?? false;
```

Comment:
Can we just infer this from this.layout == GroupedLayout?

Reply:
No really, layout and group nodes are two different things